CHANGELOG.md

```diff
@@ -6,6 +6,17 @@ Changes by Version
 
 #### Backend Changes
 
+- Remove cassandra SASI indices [#1328](https://github.com/jaegertracing/jaeger/pull/1328)
+
+Migration Path:
+
+1. Run `plugin/storage/cassandra/schema/migration/v001tov002part1.sh` which will copy dependencies into a csv, update the `dependency UDT`, create a new `dependencies_v2` table, and write dependencies from the csv into the `dependencies_v2` table.
+2. Run the collector and query services with the cassandra flag `enable-dependencies-v2=true` which will update jaeger to write and read to and from the new `dependencies_v2` table.
+3. Update [spark job](https://github.com/jaegertracing/spark-dependencies) to write to the new `dependencies_v2` table.
```

Comment:
this should rather say "update Spark job to version N"

Reply:
I'll punt on this until I've actually made the change in spark.